src/command_modules/azure-cli-cosmosdb/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -90,3 +90,5 @@ def validate_ip_range_filter(ns):
                       options_list=('--indexing-policy'),
                       help='Indexing Policy, you can enter it as a string or as a file, e.g., --indexing-policy @policy-file.json)',
                       type=shell_safe_json_parse, completer=FilesCompleter())
+
+register_cli_argument('cosmosdb collection', 'defaultTtl', options_list=('--default-ttl'), help='Default TTL', type=int)
```

Comment:
Since the option name is the same as the underlying parameter name, you can omit it this kwarg.  "default_ttl" will automatically be translated to "--default-ttl".